command/bridge/deposit/deposit.go

```diff
@@ -0,0 +1,295 @@
+package deposit
+
+import (
+	"fmt"
+	"math/big"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/umbracle/ethgo"
+	"golang.org/x/sync/errgroup"
+
+	"github.com/0xPolygon/polygon-edge/command"
+	"github.com/0xPolygon/polygon-edge/command/rootchain/helper"
+	"github.com/0xPolygon/polygon-edge/consensus/polybft"
+	"github.com/0xPolygon/polygon-edge/consensus/polybft/contractsapi"
+	"github.com/0xPolygon/polygon-edge/consensus/polybft/contractsapi/artifact"
+	"github.com/0xPolygon/polygon-edge/contracts"
+	"github.com/0xPolygon/polygon-edge/txrelayer"
+	"github.com/0xPolygon/polygon-edge/types"
+)
+
+type TokenType int
+
+const (
+	ERC20 TokenType = iota
+	ERC721
+	ERC1155
+)
+
+const (
+	// defaultMintValue represents amount of tokens which are going to be minted to depositor
+	defaultMintValue = int64(1e18)
+)
+
+var (
+	params depositParams
+
+	manifest *polybft.Manifest
+
+	tokenTypesMap = map[string]TokenType{
+		"erc20":   ERC20,
+		"erc721":  ERC721,
+		"erc1155": ERC1155,
+	}
+
+	configs = map[TokenType]*bridgeConfig{}
+)
+
+// GetCommand returns the bridge deposit command
+func GetCommand() *cobra.Command {
+	depositCmd := &cobra.Command{
+		Use:     "deposit",
+		Short:   "Deposits tokens from root chain to child chain",
+		PreRunE: runPreRun,
+		Run:     runCommand,
+	}
+
+	setFlags(depositCmd)
+
+	return depositCmd
+}
+
+func setFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVar(
+		&params.manifestPath,
+		manifestPathFlag,
+		"./manifest.json",
+		"the manifest file path, which contains genesis metadata",
+	)
+
+	cmd.Flags().StringVar(
+		&params.tokenTypeRaw,
+		tokenFlag,
+		"erc20",
+		"token type which is being deposited",
+	)
+
+	cmd.Flags().StringSliceVar(
+		&params.receivers,
```

Comment:
We could consider merging the 'receivers' and 'amount' elements flags into a single flag that contains both the recipient's address and their corresponding payment amount, like this: address:amount. However, in the context of the current pull request, there is no need to make this change.

Reply:
Probably going to change it sometime in the future.